source/_docs/terminus/plugins/create.md

```diff
@@ -0,0 +1,157 @@
+<figure><pre id="terminus-plugin-namespace"><code class="bash" data-lang="bash">namespace Pantheon\TerminusHelloWorld\Commands;</code></pre></figure>
+</div>
+The `Commands` part of the namespace is not strictly necessary but it can help keep things organized if you need to add supporting classes to your plugin.
+
+You will also need to make sure your src directory and composer file reflect the new namespace. Move the `HelloCommand.php` file  from `src/` to the `src/Commands` directory to mirror the last part of the namespace. If you have a lot of commands in your plugin, you can group them into command groups by adding another layer to the namespace and directory structure.
+
+Update the `composer.json` file with an autoload section to indicate how to load your namespace. Your composer file should now look like:
+```
+{
+    "name": "pantheon-systems\terminus-hello-world",
+    "description": "An Hello, World Terminus command",
+    "type": "terminus-plugin",
+    "autoload": {
+        "psr-4": { "Pantheon\\TerminusHello": "src" }
+    },
+    "extra": {
+        "terminus": {
+            "compatible-version": "1.*"
+        }
+    }
+}
+```
+Remember to change `pantheon-systems` and `Pantheon` in the above to your vendor name.
+
+## Coding Standards
+While not strictly necessary, if you plan to distribute your plugin, and especially if you plan to add an open source licence and encourage contributions, it may be a good idea to adopt the Terminus core standards. Some basics to follow are:
+
+* Ensure for compatibility with PHP 5.5, 5.6 and 7.
+* Follow [PSR-2 code style](http://www.php-fig.org/psr/psr-2/).
+
+More information on Terminus standards can be found at:
+[https://github.com/pantheon-systems/terminus/blob/master/CONTRIBUTING.md](https://github.com/pantheon-systems/terminus/blob/master/CONTRIBUTING.md)
+
+## Plugin Versioning
+Terminus follows semantic versioning. That means there should not be any plugin-breaking changes within a major version of Terminus but plugins will not be compatible between major versions of Terminus. If distribute your plugin, please make sure that the major version of your plugin matches the major version of Terminus that it is compatible with. This may force you to make backwards-incompatible changes in a minor release, but it will reduce confusion among plugin users (who will be assured that a 1.x version of a plugin will work with a 1.x version of Terminus). If your plugin has a minimum required version of Terminus, you can specify that in the `compatible-version` section of `composer.json`. You can use the [standard composer version constraints syntax](https://getcomposer.org/doc/articles/versions.md). If you do change `compatible-version`, please make sure that your constraint expression does not accidentally include the next major version of Terminus. In other words, `>=1.3 <2.0.0` is fine but `>=1.3` is not.
```

Comment:
Perhaps instead of saying `This may force you to make backwards-incompatible changes in a minor release,`, we should instead advise them to use:

TERMINUS.MAJOR.MINOR

So the first digit of their version is always the Terminus major version the plugin is compatible with, the second digit is advanced for breaking changes, and the final digit is used for minor or patch (bugfix) releases.